src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkGrids/ArtistArtworkGrid.tsx

```diff
@@ -42,6 +45,13 @@ export const ArtistArtworkGridFragmentContainer = createFragmentContainer<{
           exclude: $excludeArtworkIDs
         ) {
           ...ArtworkGrid_artworks
+
+          # Used to check for content
+          edges {
```

Comment:
#sominor some of our connections in MP have `totalCount` added as well to the connection (https://github.com/artsy/metaphysics/blob/a1561b1456667b50a93c2799b7f8ee13a8ee347e/src/schema/fields/pagination.ts#L154)

Since these get all collapsed with the `...ArtworkGrid_artworks` fragment I don't think this adds _any_ extra data to the request (it's just maybe a bit annoying/ugly to keep repeating it just for existential checks), but at some point if we make sure `totalCount` exists on all connections we could just use that (which may read better).

Reply:
Ah nice -- I'll follow up and see if this exists on these grids and will swap out 👍